extensions/cpf-cnpj-generator/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "cpf-cnpj-generator",
+  "title": "CPF/CNPJ Generator",
+  "description": "Generate CPF or CNPJ data for developers",
```

Comment:
I'm wondering if we could make it clearer what `CPF` and `CNPJ` are about? Like `CPF` means something else in french so it might be confusing. Maybe write it down in full here?

Reply:
@mathieudutour I just updated the description clarifying what CPF and CNPJ mean. Can you take a look?

Reply:
Looks good :+1: